src/components/ServiceRequestForm.jsx

```diff
@@ -154,33 +174,21 @@ const ServiceRequestForm = (props, errors, touched) => {
 		}
 
 		pullServiceRequestFields(fields);
-	};
-
-	const pullServiceRequestFields = (fields) => {
-		if (fields !== undefined) {
-			localProps.setFieldValue('requestTypeAddressID', fields.streetAddress);
-			localProps.setFieldValue('requestTypeCityID', fields.city);
-			localProps.setFieldValue('requestTypeZipID', fields.zipCode);
-			localProps.setFieldValue('isPanelRequired', true);
-		}
-		else{
-			localProps.setFieldValue('isPanelRequired', false);
-		}
-	};
-
-	const handleServiceSubRequestChange = (changeEvent) => {
+	}
 
-		const value = changeEvent.currentTarget.value.toLowerCase();
-		const subInfo = getSubCategoriesIncludedDescription(subCategories, value ? value : value);
+	const addSelectedSubValueOptions = (Categories, value)=>{
+		const subCategories = Categories.flatMap(x => x.types);
```

Comment:
I think the notes field is working (which uses this) so i think this is ok, but we just want to make sure this is polyfilled in IE.